setup.py

```diff
@@ -17,7 +17,7 @@
     keywords=["django", "notification" "alerts"],
     packages=find_packages(),
     zip_safe=False,
-    install_requires=["django>=1.11,<2.1"],
+    install_requires=["django>=1.11,<2.1.1"],
```

Comment:
This allows version 2.1, which isn't tested, nor in the test matrix. The change would require to edit `tox.init` and `travis.ini` and should instead define a new upper bound as `2.2`.

Reply:
okay think I have made the changes as described, if they are good and tests pass I will do the same for django-wiki